integration-cli/docker_cli_logs_test.go

```diff
@@ -178,6 +178,41 @@ func (s *DockerSuite) TestLogsSince(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestLogsUntil(c *check.C) {
```

Comment:
@jamiehannaford as @dperny said 👼 Sorry for the trouble 🙏 
I'll update the contributing docs to make it clear 👼